src/Core/tests/DeviceTests/Handlers/ScrollView/ScrollViewHandlerTests.iOS.cs

```diff
@@ -52,5 +52,38 @@ public async Task ContentInitializesCorrectly()
 
 			Assert.True(result, $"Expected (but did not find) a {nameof(MauiTextField)} in the Subviews array");
 		}
+
+		[Fact]
+		public async Task ScrollViewContentSizeSet() 
+		{
+			var scrollView = new ScrollViewStub();
+
+			var scrollViewHandler = await InvokeOnMainThreadAsync(() =>
+			{
+				return CreateHandler(scrollView);
+			});
+
+			await InvokeOnMainThreadAsync(async () => {
+				await scrollViewHandler.PlatformView.AttachAndRun(() =>
+				{
+				
+					var entry = new EntryStub() { Text = "In a ScrollView", Height = 10000 };
+					var entryHandler = Activator.CreateInstance<EntryHandler>();
```

Comment:
You can register a handler as part of the test to avoid this code

```C#
EnsureHandlerCreated(builder =>
			{
				builder.ConfigureMauiHandlers(handler =>
				{
					handler.AddHandler<EntryStub, EntryHandler>();
				});
			});
```